### DISCUSSION TITLE
Large data sets

### DISCUSSION THREAD

**Discussion by jobrow-msft** [Live-Charts/LiveCharts2#1057] [Q&A] [answered]
Hi! LiveCharts2 looks awesome and we're considering integrating it into a project but we're wondering how well it scale with large data sets, particular the graphing visualizations. Has anyone used it with large data sets (a few million points) and able to provide any insight? Thanks!

**Comment by beto-rodriguez** [comment, marked as answer]
Hi 

Tere are some tests on large data sets, that package is not yet available publicly.

https://github.com/beto-rodriguez/LiveCharts2/releases/tag/v2.0.0-beta.350

**Comment by jobrow-msft** [comment]
Nifty, thanks! It looks like that package has been in pre-release for a while. Any idea when it will be puclicly available?

**Comment by beto-rodriguez** [comment]
I hope we can make it by November of this year.

**Reply by DarkMortainius** [reply]
Any update on when this might be available in a public release? I have a project that uses about 35,000 data points, and the chart is just barely usable. Having the high performance option would be a huge help.

**Reply by kucint** [reply]
Hi, I reconsider as well to use LiveCharts in my project that will use big number of data points. 
Any update on this topic?

**Reply by Renger-Baulds** [reply]
Is there still any high performance package under development that is not included in release 2.0.0-rc2?
And if so, when will it be released and available in a NuGet package?
We run into bad UI performance in WPF with a cartesian chart using large real-time datasets.
This can easily be reproduced in your WPF sample application, General/MultiThreading, using e.g. 15k datapoints instead of 1.5k.

**Reply by BlueFrog76** [reply]
I'm guessing there are not any updates on this performance package?  I don't see any sign of the config.AddBackersPackageAndDefaults() in the rc4 release of LiveCharts2.  We routinely work with data sets that are over 10,000 points and are seeing significant performance issues with large datasets.  We would really like to use LiveCharts2, but the performance issues are a significant drawback.  The rc2 post makes it sound like significant performance improvements have been acheived with up to 10 million points providing updates in just 200 ms.  We had hoped that the package was incorporated already, but it does not appear to be based on the tests we have performed.  When will this updated performance package be available?
